packages/vscode-extension/src/debug/prerequisitesHandler.ts

```diff
@@ -1042,20 +1042,20 @@ function checkNpmInstall(
               }
             });
           } else {
-            exitCode = await runTask(
-              new vscode.Task(
-                {
-                  type: "shell",
-                  command: taskName,
-                },
-                vscode.workspace.workspaceFolders![0],
-                taskName,
-                ProductName,
-                new vscode.ShellExecution([baseNpmInstallCommand, ...args].join(" "), {
-                  cwd: folder,
-                })
-              )
+            const task = new vscode.Task(
+              {
+                type: "shell",
+                command: taskName,
+              },
+              vscode.workspace.workspaceFolders![0],
+              taskName,
+              ProductName,
+              new vscode.ShellExecution([baseNpmInstallCommand, ...args].join(" "), {
+                cwd: folder,
+              })
             );
+            task.presentationOptions.reveal = vscode.TaskRevealKind.Never;
```

Comment:
Adjust the progress bar as well:
- wording - something like "Check [terminal window]() for detailed logs..."
- Is it possible to add "Open Terminal Window" button to progress bar?
- Is it possible to expand all progress bar text by default?

Reply:
- Currently we use the progress handler from the common utils for all extension codes. It hard codes "Check **output window** for details.". Do we need to modify it or implement our own progress handler?
- vscode do not provide the API to add a button to a progress bar.
- Seems not. Do not find a way to expand the text by default.

Reply:
At least we need to tell user the detailed logs are in terminal window, so:
- If hard to replace "Check output window...", just add "Open [terminal window]()..." to the message
- Add "Open terminal window..." to output log as well
- Popup another notification (with open terminal button) together with the progress bar
- ...

Reply:
Sure. Let me update it in another PR.